package.json

```
@@ -4,7 +4,12 @@
   "description": "Custom Vue templates & resources for DigitalOcean Community tools.",
   "main": "src/cli.js",
   "scripts": {
-    "test": "eslint './src/**/*.{vue,js}'"
+    "test": "eslint './src/**/*.{vue,js}' && npm run demo:build",
```

```suggestion
    "test": "npm run test:eslint && npm run demo:build",
    "test:eslint": "eslint './src/**/*.{vue,js}'",
    "test:eslint:fix": "npm run test:eslint -- --fix",
```
Consistent with how we do these scripts elsewhere